crates/rustc_codegen_spirv/src/lib.rs

```diff
@@ -380,6 +380,9 @@ impl CodegenBackend for SpirvCodegenBackend {
     ) -> Result<(), ErrorReported> {
         // TODO: Can we merge this sym with the one in symbols.rs?
         let legalize = !sess.target_features.contains(&Symbol::intern("kernel"));
+        let emit_multiple_modules = sess
+            .target_features
+            .contains(&Symbol::intern("multimodule"));
```

Comment:
Would it be more appropriate if we accepted this as an argument from `-Cllvm-args`([`sess.cg.llvm_args`](https://doc.rust-lang.org/nightly/nightly-rustc/rustc_session/config/struct.CodegenOptions.html#structfield.llvm_args))? It's not really something that's specific to the the target as it is specific to our backend. Another benefit is since it is just `Vec<String>`, we could use clap here for getting these arguments, that's make it to have key value arguments like `--module-output=[multiple|single]`.